### ISSUE TITLE
Display 3 days in a week in mobile view

### ISSUE THREAD

**Issue by touchwithsridhar** [jquense/react-big-calendar#2197]
### Clear and concise description of the problem

Hi There,
There is a problem in mobile view. the date are not visible fully in mobile view. Can we show only 3 days in the week view in mobile view. on next click next 3 days so on..
Please help me on this

### Suggested solution

as of now week view is displaying 7 days in the header, work_week is displaying 5 days in a week. But for me i need to show only 3 days in the week view and on next button click it will be next 3 days like that.

### Validations

- [X] Read the [docs](http://jquense.github.io/react-big-calendar/examples/index.html).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by cutterbl** [comment]
You are correct. Tablet display is ok (for the most part), but it is not currently written to support small devices.
